Gopkg.toml

```diff
@@ -18,7 +18,7 @@
 [[constraint]]
   name = "google.golang.org/grpc"
   source = "https://github.com/grpc/grpc-go.git"
-  version = "1.12.0"
```

Comment:
Why do we use "source =" for this package?

Reply:
I think the reason is go-grpc's import path is different from its source path.

Reply:
The package contains all the source needed though, so you should be able to delete the "source" line.